modules/tinymce/src/core/test/ts/browser/FormatterApplyTest.ts

```diff
@@ -2308,4 +2308,182 @@ describe('browser.tinymce.core.FormatterApplyTest', () => {
     editor.formatter.apply('bold');
     assert.equal(getContent(editor), '<p><strong>test</strong></p><table><tbody><tr><td><strong>cell 1</strong></td><td>cell 2</td></tr><tr><td><strong>cell 3</strong></td><td>cell 4</td></tr></tbody></table>');
   });
+
+  it('TINY-6567: Apply format including the final bullet point in the list', () => {
+    const editor = hook.editor();
+    editor.setContent(
+      '<ul>' +
+        '<li>a</li>' +
+        '<li>b<br />' +
+          '<ul>' +
+            '<li>c</li>' +
+            '<li>d</li>' +
+          '</ul>' +
+        '</li>' +
+      '</ul>'
+    );
+    LegacyUnit.setSelection(editor, 'ul', 0, 'ul li li:nth-child(2)', 0);
```

Comment:
Siwtch to using TinySelections.setSelection here since that is used in the remove tests.

Reply:
updated